google/cloud/forseti/services/inventory/base/resources.py

```diff
@@ -2365,6 +2567,25 @@ class StorageObjectIterator(resource_iter_class_factory(
             StorageBucketIterator,
         ]}),
 
+    'kubernetes_cluster': ResourceFactory({
```

Comment:
nit: please move these down with the other kubernetes resources so they are alphabetical?

Reply:
Done